packages/labs/src/components/popover/popover2.tsx

```diff
@@ -255,7 +255,7 @@ export class Popover2 extends AbstractComponent<IPopover2Props, IPopover2State>
         const targetTabIndex = this.props.openOnTargetFocus && this.isHoverInteractionKind() ? 0 : undefined;
         const target: JSX.Element = React.cloneElement(children.target, {
             // force disable single Tooltip child when popover is open (BLUEPRINT-552)
-            disabled: isOpen && children.target.type === Tooltip2 ? true : this.props.disabled,
+            disabled: isOpen && children.target.type === Tooltip2 ? true : undefined,
```

Comment:
this isn't correct either, as it removes any value set by the user... should be `children.target.props.disabled`. or do the thing that `Popover` does where it builds the object over a few lines.

```js
// pseudo code
const props = { tabIndex };
if (is disabled) { props.disabled = true; }
cloneElement(target, props)
```